src/DIRAC/WorkloadManagementSystem/Client/JobState/JobState.py

```diff
@@ -441,3 +444,16 @@ def insertIntoTQ(self, manifest=None):
                     gLogger.info("Job %s removed from the TQ" % self.__jid)
             return S_ERROR("Cannot insert in task queue: %s" % errMsg)
         return S_OK()
+
+    def getJobPath(self):
+        """
+        Get JobPath attribute from manifest if founded, otherwise return None
+
+        :return: list[str] / None
```

Comment:
It should consistently return the same type, so either a list or S_OK/S_ERROR.

Reply:
I change the code since, this issue is solved